storage/innobase/xtrabackup/src/xbcloud/xbcloud.cc

```diff
@@ -1234,6 +1297,34 @@ int main(int argc, char **argv) {
              ->probe_api_version_and_lookup(container_name)) {
       return EXIT_FAILURE;
     }
+  } else if (opt_storage == AZURE) {
+    std::string storage_account =
+        opt_azure_account != nullptr ? opt_azure_account : "";
+
+    if (storage_account.empty() && opt_azure_development_storage) {
+      storage_account.assign(azure_development_storage_account);
+    }
+
+    std::string access_key =
+        opt_azure_access_key != nullptr ? opt_azure_access_key : "";
+    if (access_key.empty() && opt_azure_development_storage) {
```

Comment:
I don't see a validation to when we do not pass an access key nor opt_azure_development_storage

Reply:
added.